php-telegram-bot/core
# WIP: Add Database Migrations (And more features as a result)

In this PR added:

- Laravel's Database Query Builder and Migrations [QB Docs](https://laravel.com/docs/5.8/queries)  [Migrations Docs](https://laravel.com/docs/5.8/migrations)
- Laravel's Service Container [CI Docs](https://laravel.com/docs/5.8/container)
- DotEnv Integration (and the possibility to create cacheable config files like Laravel's config)
- Possibility to create CLI Commands like `php telegram blabla`

This brings only one BC change, PHP version should be >=7.1.3, but versions below 7.1 are already deprecated and not supported anymore.

@noplanman @jacklul please take a look, what do you think about the changes

How to test:

Pull this branch, copy `.env.example` to `.env` and write your settings, after run `php telegram migrate`

Two tables should be created, `migrations` for holding versions and `users` table which just copied from Laravel's default migration.
